patch/options.go

```diff
@@ -26,3 +27,19 @@ func fieldOptions(f *protogen.Field) *gopb.Options {
 func oneofOptions(o *protogen.Oneof) *gopb.Options {
 	return proto.GetExtension(o.Desc.Options(), gopb.E_Oneof).(*gopb.Options)
 }
+
+func getBoolExtension(pb proto.Message, extension protoreflect.ExtensionType, ifnotset bool) bool {
```

Comment:
Negative assertions with Boolean fields should be prefixed or marked as such. This helper should be inverted.